scripts/guests.js

```diff
@@ -10,4 +10,21 @@
                 Add park area id as state
 
         Return HTML String
-*/
\ No newline at end of file
+*/
+
+import { getGuestsList } from "./database.js"
+
+
+export const guestsList = () => {
+    const guests = getGuestsList()
+    let htmlString = `<div class="guests">`
```

Comment:
As this is the div for the entire list of guests, would it make more sense to name this class `guest-list`?